frontend/src/cards/RateTrendsChartCard.tsx

```diff
@@ -0,0 +1,161 @@
+  const metricConfigRates = props.variableConfig.metrics["per100k"];
+  const metricConfigPctShares = props.variableConfig.metrics["pct_share"];
+
+  const breakdowns = Breakdowns.forFips(props.fips).addBreakdown(
+    props.breakdownVar,
+    exclude(NON_HISPANIC)
+  );
+
+  const ratesQuery = new MetricQuery(
+    metricConfigRates.metricId,
+    breakdowns,
+    LONGITUDINAL
+  );
+  const pctShareQuery = new MetricQuery(
+    metricConfigPctShares.metricId,
+    breakdowns,
+    LONGITUDINAL
+  );
+
+  function getTitleText() {
+    return `${
+      metricConfigRates.trendsCardTitleName
+    } by ${BREAKDOWN_VAR_DISPLAY_NAMES[
+      props.breakdownVar
+    ].toLowerCase()} in ${props.fips.getSentenceDisplayName()}`;
+  }
+  function CardTitle() {
+    return <>{getTitleText()}</>;
+  }
+
+  return (
+    <CardWrapper
+      queries={[ratesQuery, pctShareQuery]}
+      title={<CardTitle />}
+      minHeight={PRELOAD_HEIGHT}
+    >
+      {([queryResponseRates, queryResponsePctShares]) => {
+        const ratesData = queryResponseRates.getValidRowsForField(
+          metricConfigRates.metricId
+        );
+
+        const pctShareData = queryResponsePctShares.getValidRowsForField(
+          metricConfigPctShares.metricId
+        );
+
+        // retrieve list of all present demographic groups
+        const demographicGroups: DemographicGroup[] =
+          queryResponseRates.getFieldValues(
+            props.breakdownVar,
+            metricConfigRates.metricId
+          ).withData;
+
+        const [knownRatesData] = splitIntoKnownsAndUnknowns(
+          ratesData,
+          props.breakdownVar
+        );
+        const [, unknownPctShareData] = splitIntoKnownsAndUnknowns(
+          pctShareData,
+          props.breakdownVar
+        );
+
+        const nestedRatesData = getNestedRates(
+          knownRatesData,
+          demographicGroups,
+          props.breakdownVar,
+          metricConfigRates.metricId
+        );
+        const nestedUnknownPctShareData = getNestedUnknowns(
+          unknownPctShareData,
+          metricConfigPctShares.metricId
+        );
+
+        return (
+          <CardContent>
+            {queryResponseRates.shouldShowMissingDataMessage([
+              metricConfigRates.metricId,
+            ]) || nestedRatesData.length === 0 ? (
+              <>
+                <MissingDataAlert
+                  dataName={`historical data for ${metricConfigRates.fullCardTitleName}`}
+                  breakdownString={
+                    BREAKDOWN_VAR_DISPLAY_NAMES[props.breakdownVar]
+                  }
+                  fips={props.fips}
+                />
+              </>
+            ) : (
+              <>
+                {/* 2N INCIDENCE RATE TRENDS VIZ COMPONENT HERE */}
+                {console.log("KNOWN RATES", nestedRatesData)}
+                {console.log("UNKNOWN PCT SHARE", nestedUnknownPctShareData)}
+                {console.log("CONFIG", BREAKDOWN_VAR_DISPLAY_NAMES)}
```

Comment:
Are these logs and comments still meant to be here?

Reply:
removed